### ISSUE TITLE
Add rageshake feature

### ISSUE THREAD

**Issue by notramo** [syphon-org/syphon#445]
If you shake your phone when Element is open, it displays a dialog about bug report.

The same could be implemented in Syphon for easy error reporting.

The Element implementation posts the reports to a HTTPS server, but in Syphon, a report file could be sent in an E2EE DM to @ereio. This way, it would be possible to say thanks for the report, and ask more questions if needed.

The dialog should say this:
> It seems like you are angrily shaking your phone. Have you found a bug?
Would you like to send a bug report? It would really help us fixing it. It will be sent in an encrypted Matrix message to Taylor Ereio, who programmed this app, and would love to hear your feedback.

If the user clicks Yes, then the app should open an encrypted DM with @ereio, and send the file.

**Comment by ereio** [comment]
Sounds great, but let's send it to #syphon-support instead ;)

**Comment by notramo** [comment]
No, because it contains details (logs, screenshots) that shouldn't be public. For example, if a screenshot contains details about contacts, that shouldn't be posted in a public room, where hundreds of non-developers can see it.That's why it would be an encrypted DM sent to only one developer. Then the developer can invite the user to #syphon-support.

**Comment by ereio** [comment]
Ah, didn't read the description as carefully. It would be a E2EE DM, probably to me at first - like you mentioned - but i would like to consider a mod or "core" team room for triage in the future.

**Comment by EdGeraghty** [comment]
One thought is to make the rageshake toggle-able, probably default-on (as it should ask permission before doing anything).

"Traditionally", a shake is the Ripple trigger so we need to balance how this would work with #332.
